src/EditorFeatures/Test/Diagnostics/TestDiagnosticAnalyzerService.cs

```diff
@@ -68,6 +68,14 @@ public override string FullPath
                 }
             }
 
+            public override string Id
+            {
+                get
+                {
+                    return nameof(TestAnalyzerReferenceByLanguage);
```

Comment:
This is just as good as what we had before, when we were simply using `DisplayName`. I don't want to implement solutions for problems we don't have yet (especially in test code).